x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/DenseVectorFieldTypeIT.java

```diff
@@ -32,7 +36,7 @@
 
 public class DenseVectorFieldTypeIT extends AbstractEsqlIntegTestCase {
 
-    private static final Set<String> DENSE_VECTOR_INDEX_TYPES = Set.of(
+    public static final Set<String> ALL_DENSE_VECTOR_INDEX_TYPES = Set.of(
```

Comment:
Would it make sense to use `VectorIndexType.values()` here to create the set, rather than hard coding the specific index types?